docs/overview/animation.md

```diff
@@ -104,6 +104,27 @@ The speed is interpreted in _unit/sec_ dimension. For example,  `lv_anim_speed_t
 
 You can delete an animation with `lv_anim_del(var, func)` if you provide the animated variable and its animator function.
 
+## Timeline
+Timeline is a collection of multiple Animations, which makes it easy to create complex composite animations. 
+
+It supports forward and reverse playback of the entire animation group, using`lv_anim_timeline_start(anim_timeline, playback)`.
+
+Use the `lv_anim_timeline_set_progress(anim_timeline, progress)` function to set the state of the object corresponding to the progress of the timeline.
+
+![](/misc/anim-timeline.png "timeline diagram")
+
+**Note**
+* All the timeline member parameters must be set. It includs:
+  1) Start time (must >= 0)
+  2) Variable to animate
+  3) Function to execute to animate
+  4) Start value
+  5) End value
+  6) Animation duration
+  7) Animation path
+  8) Early apply (When the timeline appears at different time points on an object and the same execution function is set, the **early_apply** parameter needs to be set to **false** to prevent the previous animation from being deleted.)
```

Comment:
@kisvegabor 
Is the explanation of **Early apply** reasonable?

Reply:
I'd say: 
"...to prevent the previous animation from being overwritten.)"

But I've just realized a potential problem. LVGL prevents adding multiple animations to the same variable with the same `exec_cb` to prevent running 2 animations on the same property at the same time. It seems this behavior can be improved like:
- Do not delete similar animation in `lv_anim_start`
- In `anim_timer` when an animation starts, check if there is a similar running animation. If so delete the other animation.

What do you think?

Reply:
I think **early_apply** should be invalidated when there is a delay in the animation, and don’t delete similar animations.

Reply:
That's much simpler. It can be even the user's responsibility as you stated i nthe docs.